rust-runtime/aws-smithy-http-server-python/src/server.rs

```diff
@@ -460,4 +455,40 @@ event_loop.add_signal_handler(signal.SIGINT,
         self.block_on_rust_signals();
         Ok(())
     }
+
+    /// Lambda main entrypoint: start the handler on Lambda.
+    ///
+    /// Unlike the `run_server`, `run_lambda_handler` does not spawns other processes,
+    /// it starts the Lambda handler on the current process.
+    fn run_lambda_handler(&mut self, py: Python) -> PyResult<()> {
+        let event_loop = self.configure_python_event_loop(py)?;
+        let app = self.build_and_configure_router(py, event_loop)?;
+        let rt = runtime::Builder::new_multi_thread()
+            .enable_all()
+            .build()
+            .expect("Unable to start a new Tokio runtime for this process");
+        rt.block_on(async move {
+            let handler = LambdaHandler::new(app);
+            let lambda = lambda_http::run(handler);
+            tracing::debug!("Starting Lambda handler");
```

Comment:
I think the general consensus in Rust ecosystem is to use lowercase log messages.

https://docs.rs/env_logger/latest/env_logger/#example
https://github.com/rust-lang/rust/issues/442

Reply:
I've changed messages introduced in this PR and created https://github.com/awslabs/smithy-rs/issues/1838 for the rest